test_pytest_timeout.py

```diff
@@ -603,3 +608,32 @@ def test_foo():
             "pytest_timeout_cancel_timer",
         ]
     )
+
+
+def test_session_timeout(pytester):
+    # 2 tests, each with 0.5 sec timeouts
+    # each using a fixture with 0.5 sec setup and 0.5 sec teardown
+    # So about 1.5 seconds per test, ~3 sec total,
+    # Therefore:
+    # A timeout of 1.25 sec should happen during the teardown of the first test
```

Comment:
0.25 from an edge is too brittle. experience tells me you can't really work with sub-seconds times in this test suite, it gets flaky otherwise.

Make the total test duration 2s and the session timeout 1s?

Reply:
All the numbers in this description are now wrong.  maybe just describe that this is designed to timeout during the first test and ensure the first test still runs to completion but the 2nd test is not started?